### PR TITLE
Test 'openio-admin <item> check'

### PR DESCRIPTION
**Pull request by AymericDu**
##### SUMMARY

- Test 'openio-admin account check'
- Test 'openio-admin container check'
- Test 'openio-admin object check'
- Test 'openio-admin chunk check'
- Allow to recover object meta when checking a missing chunk

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

- Admin CLI

##### SDS VERSION

```
openio 5.0.0.0b1.dev13
```

### CODE REVIEW

File: oio/crawler/integrity.py [hunk 1]
```diff
@@ -210,6 +213,28 @@ def complete_target_from_chunk_metadata(self, target, xattr_meta):
                                      "and container names: %s",
                                      cid, err)
 
+    def recover_and_complete_object_meta(self, target, chunk):
+        _, rawx_service, chunk_id = chunk.rsplit('/', 2)
+        # 1. Fetch chunk list from rdir (could be cached).
+        # Unfortunately we cannot seek for a chunk ID.
```
Comment (murlock) [inline oio/crawler/integrity.py]: Is it possible to use `start_after` with value `chunk_id - 1`?

File: tests/functional/cli/admin/test_item_check.py [hunk 2]
```diff
@@ -0,0 +1,1357 @@
...
+        opts = self.get_opts(['Id'])
+        output = self.openio(
+            '--oio-account %s object locate %s --auto --flat-bits %d %s'
+            % (account, obj_name, self.FLAT_BITS, opts))
+        obj_chunks = output.rstrip('\n').split('\n')
+        container = self.autocontainer(obj_name)
+        self._wait_events(account, container, obj_name)
+        return container, obj_meta, obj_chunks
+
+    def corrupt_chunk(self, chunk):
```
Comment (murlock) [inline tests/functional/cli/admin/test_item_check.py]: Should we add also `corrupt_chunk_xattr` ?
(Maybe it is done in another PR ?)

File: oio/crawler/integrity.py (hunk 1 above)
Comment (AymericDu) [reply oio/crawler/integrity.py]: We can't use `start_after`... This parameter uses this format: `CONTAINER_ID|CONTENT_ID|CHUNK_ID`. And we don't know `CONTAINER_ID` and `CONTENT_ID` (because that's what we're looking for).

File: tests/functional/cli/admin/test_item_check.py (hunk 2 above)
Comment (AymericDu) [reply tests/functional/cli/admin/test_item_check.py]: I agree we are doing this test in another PR.

File: oio/crawler/integrity.py (hunk 1 above)
Comment (fvennetier) [reply oio/crawler/integrity.py]: In the future we can implement a route to make this filtering server-side.
